crates/rpc/rpc-types/src/serde_helpers/u64_hex.rs

```diff
@@ -16,3 +16,18 @@ where
 pub fn serialize<S: Serializer>(value: &u64, s: S) -> Result<S::Ok, S::Error> {
     U64::from(*value).serialize(s)
 }
+
+/// serde functions for handling `Option<u64>` as [U64]
+pub mod option_u64_hex {
```

Comment:
```suggestion
pub mod u64_hex_opt {
```

Reply:
addressed in https://github.com/anton-rs/op-reth/pull/133